### PR TITLE
release 1.2.0

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -10,3 +10,5 @@ Matthias Dellweg <2500@gmx.de> <mdellweg@redhat.com>
 Matthias Dellweg <2500@gmx.de> <dellweg@atix.de>
 Mark Hlawatschek <hlawatschek@atix.de> <mark@atix.de>
 Paul Gration <pmgration@gmail.com>
+Peter Ondrejka <pondrejk@redhat.com> <pondrejk@users.noreply.github.com>
+Richard <richielatk@gmail.com> <stempfl@atix.de>"
```
Comment (mdellweg) [inline .mailmap]:
```suggestion
Richard Stempfl <richielatk@gmail.com> <stempfl@atix.de>"
```

Comment (mdellweg) [reply .mailmap]: Or isn't that how the mailmap works?

Comment (evgeni) [reply .mailmap]: that works, yeah. he never had his last name in the commits, that's why I ommited it originally.

File: galaxy.yml [hunk 2]
```diff
@@ -38,7 +38,7 @@ authors:
   - "Patrick Creech <pcreech@redhat.com>"
   - "Paul Armstrong <parmstro@users.noreply.github.com>"
   - "Paul Gration <pmgration@gmail.com>"
-  - "Peter Ondrejka <pondrejk@users.noreply.github.com>"
+  - "Peter Ondrejka <pondrejk@redhat.com>"
   - "Philipp <philipp98.joos@gmail.com>"
   - "Quirin Pamp <Q.Pamp.2009@my.bristol.ac.uk>"
   - "Richard <richielatk@gmail.com>"
```
Comment (mdellweg) [inline galaxy.yml]:
```suggestion
  - "Richard Stempfl <richielatk@gmail.com>"
```

**Review by mdellweg — APPROVED** [review]

File: .mailmap (hunk 1 above)
Comment (evgeni) [reply .mailmap]: updated

Comment (mdellweg) [reply .mailmap]: should we have his consent?

@IncredibleRichie

Comment (IncredibleRichie) [reply .mailmap]: It´s okay for me. :-)

**Pull request merged**